src/walter_data/WALTer.lpy

```diff
@@ -2347,17 +2351,22 @@ def EndEach(lstring, lscene):
                     Debug_PAR_dico_df["tiller_surface"].append(tiller_surface[(lstring[id][0].num_plante, lstring[id][0].tiller)]*1e-4)
                     Debug_PAR_dico_df["Inc_PAR"].append(current_PAR)
 
-          if write_output_file["PAR_per_axes"] == True:
-            for num_plt in axis_census.keys():
-              for axis in axis_census[num_plt].keys():
-                PAR_per_axes_dico["Elapsed_time"].append(elapsed_time)
-                PAR_per_axes_dico["Temp_cum"].append(Tempcum)
-                PAR_per_axes_dico["Num_plante"].append(num_plt)
-                PAR_per_axes_dico["Num_talle"].append(axis)
-                PAR_per_axes_dico["Inc_PAR"].append(current_PAR)
-                tt = round(Tempcum, 1)
-                PAR_per_axes_dico["Sum_PAR"].append(dico_PAR_per_axis[num_plt][axis].get(tt,0.))
-                PAR_per_axes_dico["Abs_int_PAR"].append(dico_absolute_PAR_per_axis[num_plt][axis].get(tt,0.))
+            par_per_axe = pd.DataFrame(PAR_per_organ).groupby(['Num_plante', 'Num_talle']).agg({'Organ_PAR': 'sum','Organ_surface': 'sum', 'tiller_surface':'mean'})
+            for (nump, numt), row in par_per_axe.iterrows():
+              dico_PAR_per_axis[nump][numt][round(Tempcum, 1)] = row.Organ_PAR / Temperature / row.Organ_surface
```

Comment:
This line is not clear for me. Does that mean :
row.Organ_PAR * row.Organ_surface / Temperature

Reply:
To me, it means : (row.Organ_PAR / Temperature) / row.Organ_surface ; so the result is in µmolPAR.degree_day-¹.cm-²